python/pynessie/_endpoints.py

```diff
@@ -180,13 +188,25 @@ def list_logs(
     :param max_result_hint: hint for the server, maximum number of results to return
     :param page_token: the token retrieved from a previous page returned for the same ref
     :param ssl_verify: ignore ssl errors if False
+    :param author: The author of a commit. This is the original committer
```

Comment:
The comments on `TreeApi` about the parameter docs apply here as well.

Reply:
fixed